nimble.bash-completion

```diff
@@ -0,0 +1,40 @@
+# vim: filetype=sh
+
+_nimble()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
+    COMPREPLY=()
+    _init_completion || return
```

Comment:
This line gives an error on my system (OSX). I simply removed it.. Is this working on Linux?